### PR TITLE
Changes are done 1. Comment out those xml records which creates error…

### PR DESCRIPTION
**Pull request by inovtec-solutions**
…s, which will be resolved later, 2. School Fee Module Updated..

### CODE REVIEW

**Comment by inovtec-solutions** [comment]
Please review and merge the code.

**Comment by JayVora-SerpentCS** [comment]
@AnuKPatel

File: exam/demo/exam_demo.xml [hunk 1]
```diff
@@ -181,6 +182,8 @@
             <field name="exam_date" eval="time.strftime('%Y')+'/08/20'"/>
             <field model="subject.subject" name="subject_id" search="[('name', '=', u'Science')]"/>
         </record>
+         -->
+
```
Comment (JayVora-SerpentCS) [inline exam/demo/exam_demo.xml:186]: explain the reason to comment. Rather, fix it.

File: school/models/student.py [hunk 2]
```diff
@@ -244,7 +244,23 @@ def check_current_year(self):
                                      compute="_compute_teacher_user",
                                      )
     active = fields.Boolean(default=True)
-
+    
+    # image: all image fields are base64 encoded and PIL-supported
+    image = fields.Binary(
```
Comment (JayVora-SerpentCS) [inline school/models/student.py:249]: I consider the method _default_image exists. I dont see in PR

File: exam/demo/exam_demo.xml (hunk 1 above)
Comment (AnuKPatel) [reply exam/demo/exam_demo.xml:186]: These issues are fixed after my PR are merged.

File: school/models/student.py [hunk 3]
```diff
@@ -244,7 +244,23 @@ def check_current_year(self):
...
+        "Photo", default=_default_image, attachment=True,
+        help="This field holds the image used as photo for the employee, limited to 1024x1024px.")
+    image_medium = fields.Binary(
```
Comment (AnuKPatel) [inline school/models/student.py:252]: **image_small** and **image_medium** should be **compute** fields which will resize the image uploaded in **photo** field.

File: school_fees/models/school_fees.py [hunk 4]
```diff
@@ -151,6 +153,11 @@ class StudentFeesStructureLine(models.Model):
     currency_id = fields.Many2one('res.currency', 'Currency')
     currency_symbol = fields.Char(related="currency_id.symbol",
                                   string='Symbol')
+    product_id = fields.Many2one('product.product', string='Product')
```
Comment (AnuKPatel) [inline school_fees/models/school_fees.py:156]: We have not considered each fee's type as a Product.

**Comment by JayVora-SerpentCS** [comment]
Closing this PR, @inovtec-solutions you have remake please.

**Pull request closed without merging**